darts/models/forecasting/regression_model.py

```diff
@@ -325,8 +325,68 @@ def _create_lagged_data(
             multi_models=self.multi_models,
         )
 
+        training_samples = self._add_static_covariates(target_series, training_samples)
+
         return training_samples, training_labels
 
+    def _add_static_covariates(self, series, features):
```

Comment:
Should this function maybe live in `darts.utils.data.tabularization.py` ? WDYT?

Reply:
yes, we can define it in `darts.utils.data.tabularization.py` since it is part of the features table generation.